script.js

```diff
@@ -0,0 +1,57 @@
+$(document).ready(function() {
+    $('#action').click(function() {
+    //DO NOT CHANGE CODE BELOW
+        var num1 = parseInt($('#num1').val());
+        var num2 = parseInt($('#num2').val());
+        //clear();
+        addNumbers(num1, num2);
+        var result = multiplyNumbers(num1, num2);
+        //$('#result-mult').text(result);
+    //DO NOT CHANGE CODE ABOVE
+
+    });
+
+    /*
+     Below this comment, create a function 
+     named addNumbers, which accepts two parameters.
+     The function should add the two parameters together
+     and write the result to the element with the id
+     result-add
+    */
+ })
+function addNumbers (num1, num2) {
+    var retVal = num1 + num2;
+    $("#result-add").html(retVal);
+}
+
+/*
+	 Below this comment, create a function
+	 named multiplyNumbers, which accepts two parameters.
+	 The function should multiply the two parameters together
+	 and return the result.
+    */
+	
+	
+	
+
+function multiplyNumbers (num1, num2) {
+	var myResult = num1 * num2;
+	$("#result-multiply").html (myResult);
```

Comment:
this function should multiply the numbers and **return** the result - not write it to #result-mult. I see you commented out the line in the original function that wrote to that id. So - instead of line 39 - you should have:
return myResult;
And let the main function do the writing.